pages/Types_of_Cross-Site_Scripting.md

```diff
@@ -16,13 +16,13 @@ This article describes the many different types or categories of
 cross-site scripting (XSS) vulnerabilities and how they relate to each
 other.
 
-Early on, two primary types of [XSS](XSS "wikilink") were identified,
+Early on, two primary types of [XSS](../attacks/Cross-site-Scripting-(XSS)) were identified,
```

Comment:
This doesn't exist:
https://owasp.org/www-community/attacks/Cross-site-Scripting-(XSS)

Reply:
It's https://owasp.org/www-community/attacks/xss/ due to the permalink in it's front mater.